pre_commit_hooks/no_commit_to_branch.py

```diff
@@ -0,0 +1,32 @@
+from __future__ import print_function
+
+import argparse
+import sys
+
+from pre_commit_hooks.util import cmd_output
+
+
+def is_on_branch(protected):
+    retval = False
+    branch = cmd_output('git', 'symbolic-ref', 'HEAD')
+    chunks = branch.strip().split('/')
+    if chunks[2] == protected:
+        retval = True
+    return retval
```

Comment:
this `if ...: return True return False` can just be `return chunks[2] == protected`